.env.example

```diff
@@ -0,0 +1,2 @@
+REACT_APP_MY_ACCOUNT_ID=
```

Comment:
Add an explanation of this two variables and what they are used for, guess comment above each would be enough